### PR TITLE
Update buster & parallel tests

### PR DESCRIPTION
**Pull request by hugoShaka**
### Description
Please explain the changes you made here.

Fixes #<ISSUE_NUMBER>

### Checklist
- [ ] Molecule tests are passing
- [ ] Extended the README / documentation, if necessary
- [ ] Test for the feature added

### CODE REVIEW

File: Vagrantfile [hunk 1]
```diff
@@ -2,54 +2,15 @@
...
-      ansible.limit = "all"
-      ansible.playbook = "test/tester-vagrant.yml"
-      ansible.become = true
-      ansible.groups = ansible_groups
-    end
+  config.vm.define "docker" do |subconfig|
+    subconfig.vm.hostname = "docker-vm"
     subconfig.vm.provider "virtualbox" do |vb|
-      subconfig.vm.network "private_network", ip: "10.0.0.50"
+      vb.memory=8192
```
Comment (toadjaune) [inline Vagrantfile:11]: Whoa, do we really need that much ram/cpus ?

File: ansible.cfg [hunk 2]
```diff
@@ -1,17 +0,0 @@
...
-display_skipped_hosts = False
-
-# Do not use cowsay
-nocows = 1
-
-# Connect to up to 50 server in parallel
-forks = 50
-
-# Speed improvement, but may break sudo with requiretty
-pipelining = True
```
Comment (toadjaune) [inline ansible.cfg:14]: Why ?

Comment (hugoShaka) [reply ansible.cfg:14]: Not used anymore. Molecule manages its own ansible.cfg. we can pass params through molecule.yaml files

File: Vagrantfile (hunk 1 above)
Comment (hugoShaka) [reply Vagrantfile:11]: Not really, tests are suceeding on 2gio with 2 full mailservers + 1 db + 1 client doing lots of calls.

On my local setup I don't like to wait

Comment (toadjaune) [reply Vagrantfile:11]: Hmmm ok for CPUs.
RAM however, if almost none of it ends up used, I'd scale it down.

(yes, that's bikeshedding)

Comment (hugoShaka) [reply Vagrantfile]: updated

**Pull request merged**